.github/workflows/release.yml

```diff
@@ -31,7 +32,12 @@ jobs:
         run: python -m pip install build twine
 
       - name: Create release
-        run: ~/auto shipit -vv
+        run: |
+          if [ "${{ github.event_name }}" = workflow_dispatch ]
+          then opts=
+          else opts=--only-publish-with-release-label
```

Comment:
You also need to remove `"onlyPublishWithReleaseLabel": true` from `.autorc`.

Reply:
thanks! now this makes more sense to me ;-)